helm.tf

```diff
@@ -83,7 +66,7 @@ resource "helm_release" "metrics_server" {
   namespace  = "kube-system"
   repository = "https://charts.bitnami.com/bitnami"
   chart      = "metrics-server"
-  version    = "5.8.11"
+  version    = "5.11.3"
```

Comment:
We should allow chart version selection (module input)

Reply:
I'm marking this as resolved because we agreed to don't do it